wotw_seedgen/areas.wotw

```diff
@@ -5553,11 +5566,11 @@ anchor InnerWellspring.DrainAreaEntrance at -1128, -3968:  # On the wall after d
       Water, Bash, Grenade=1, Grapple OR Glide
       Launch
     gorlek:
-      DoubleJump, Water OR TripleJump OR Dash OR Bash OR Glide OR Sword OR Hammer OR Damage=20
```

Comment:
It seems to me like Damage=20 was used as a substitute for Water here.
If that's correct, removing Water without removing Damage=20 doesn't seem logical, since Damage boosting on the Water is harder than swimming through it.

Reply:
`DoubleJump, Water` is already a path in Moki (line 5564), so I removed it because it was redundant.